Atomic/mount.py

```diff
@@ -196,8 +196,11 @@ def mount(self, best_mountpoint_for_storage=False):
 
         elif self.storage.lower() == "ostree":
             try:
-                if self._try_ostree_mount(best_mountpoint_for_storage):
-                    return
+                res = self._try_ostree_mount(best_mountpoint_for_storage)
+                # If ostree storage was explicitely requested, then we have to
+                # error out if the container/image could not be mounted.
+                if res == False:
+                    raise ValueError("Could not mount {}".format(self.image))
```

Comment:
that is the exception type that can be raised from the OSTree bindings.  In this case OSTreeMount, both as a fallback and in the case the user != root, tries a checkout from OSTree and it can fail with that exception.

We cannot really use "Image not found" as the same mount is use to mount a container (that might be using an already deleted image from the repository), so I preferred the more generic "Cannot mount X", where X can be both an image or a container.

Reply:
>that is the exception type that can be raised from the OSTree bindings

ah I see, I think it makes sense to me now =). 

> We cannot really use "Image not found" as the same mount is use to mount a container

Sorry for not being clear for the first time =(. I was trying to say that .. the source of this failure( FALSE )  was because the identifier(name) was neither found in containers or in image backend right? The `Cannot mount X` failure seems to me that the cause of failure was due to a mounting error? But it is really a minor nit =), so I think it is ok to keep the description.